src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSR.java

```diff
@@ -57,6 +58,9 @@ public void onTraversalStart() {
         outputWriter = new SAMFileGATKReadWriter(new SAMFileWriterFactory().makeWriter(outputHeader, true, OUTPUT, referenceArguments.getReferenceFile()));
         transform = new BQSRReadTransformer(outputHeader, BQSR_RECAL_FILE, bqsrArgs);
         Utils.warnOnNonIlluminaReadGroups(getHeaderForReads(), logger);
+        if (! bqsrArgs.disableIndelQuals) {
+            throw new UserException.BadArgumentValue("disable_indel_quals", "false", "GATK4 does not support BQSR on indels");
```

Comment:
I vote for removing the argument and breaking backwards compatibility rather than forcing people to always pass a pointless flag.